### PR TITLE
Feature tournament sponsors

### PR DESCRIPTION
**Pull request by bramvankooten**
Add information of tournament sponsors and prizes

### CODE REVIEW

File: src/lancie-sponsors/lancie-sponsors.html [hunk 1]
```diff
@@ -49,6 +49,20 @@ <h3>Premium sponsors</h3>
           <a href="https://www.netcompany.com/" target="_blank" rel="noopener noreferrer"><img src="images-optimized/logos/Netcompany.png"></a>
         </div>
       </div>
+      <div class="vertical-flex">
+        <h3>Supporting partners</h3>
+        <div >
+          <a href="https://www.coolermaster.com/" target="_blank" rel="noopener noreferrer"><img src="images-optimized/logos/Cooler_Master_Logo.png"></a>
+          <a href="https://www.escape-event.nl/" target="_blank" rel="noopener noreferrer"><img src="images-optimized/logos/escape-event.png"></a>
```
Comment (Sebastien1999) [inline src/lancie-sponsors/lancie-sponsors.html]: The old logo can be seen here, it should be updated to the raccoon games one.

File: data/official-tournaments.json [hunk 2]
```diff
@@ -78,30 +84,49 @@
...
     "prizes": [
-      "TBA"
+      "TrackMania Goodie bag",
+      "TrackMania Goodie bag",
+      "TrackMania Goodie bag"
+    ]
+  },
+  {
+    "title": "???",
+    "subtitle": "",
```
Comment (Sebastien1999) [inline data/official-tournaments.json]: maybe add something like "To be revealed soon" of de datum waarop het gerevealed wordt

**Comment by julian9499** [comment]
I see this 
image on google chrome. Is this just a local issue just for me?

**Comment by Sebastien1999** [comment]
The information looks good, the sponsors are shown quite well which is nice!

**Comment by bramvankooten** [comment]
> I see this
> image on google chrome. Is this just a local issue just for me?

You have to run the command `yarn run build optimize-images` first

File: data/official-tournaments.json (hunk 2 above)
Comment (bramvankooten) [reply data/official-tournaments.json]: I talked about this with Dante, it will become quite cluttered if you put something like that there

**Pull request merged**